WebRoot/js/zimbraMail/share/model/ZmSettings.js

```diff
@@ -1203,9 +1203,6 @@ ZmSettings.prototype._showConfirmDialog = function(text, callback, style) {
 
 ZmSettings.prototype._implicitChangeListener =
 function(ev) {
-	if (!appCtxt.get(ZmSetting.OPTIONS_ENABLED)) {
```

Comment:
Removing this check to apply implicit setting change listener.

is that fine @silentsakky @BhavinBathani ?

Reply:
this function does more then just setting expand/collapse status of folder, looks like we are breaking that functionality

Reply:
ok then will add check to only allow expand/collapse status of folder